gensim/models/word2vec.py

```diff
@@ -1869,3 +1873,33 @@ def __iter__(self):
         model.accuracy(args.accuracy)
 
     logger.info("finished running %s", program)
+    
+def methodize(func, instance):
+    return MethodType(func, instance, instance.__class__)
+
+
+def load(self, *args, **kwargs):
+    logger.warn('Load was called on instance. Calling on class instead')
+    Word2Vec.load(*args, **kwargs)
```

Comment:
This is not expected behaviour. 
Expected behaviour is to print warning but still run the instance method.